ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-upi-e2e.yaml

```diff
@@ -1222,10 +1222,8 @@ objects:
             echo "Removing entries from DNS ..."
             export HOSTED_ZONE_ID=$(aws route53 list-hosted-zones-by-name --dns-name "shiftstack.devcluster.openshift.com" | python -c 'import json,sys;print json.load(sys.stdin)["HostedZones"][0]["Id"].split("/")[-1]')
             sed -e's/UPSERT/DELETE/g' ${ASSETS_DIR}/api-record.json > ${ASSETS_DIR}/delete-api-record.json
-            sed -e's/Create/Delete/g' ${ASSETS_DIR}/api-record.json > ${ASSETS_DIR}/delete-api-record.json
```

Comment:
Should it have been modified to:
```sh
sed -i -e's/Create/Delete/g' ${ASSETS_DIR}/delete-api-record.json > ${ASSETS_DIR}/delete-api-record.json
```
instead of deleting the line?

Reply:
We create that Route53 change just above, and we use `UPSERT`; It shouldn't be necessary to add a second substitution.

Reply:
The second substitution was likely meant to update the comment:
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/10944/rehearse-10944-pull-ci-openshift-installer-master-e2e-openstack-upi/1295434878459842560/artifacts/e2e-openstack-upi/installer/delete-api-record.json

Reply:
that's attention to detail